acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multikeysigner/MultiKeyAcceptanceTestBase.java

```diff
@@ -86,23 +86,31 @@ public void createFileBasedTomlFileAt(
   }
 
   public void createHashicorpTomlFileAt(
-      final Path tomlPath,
-      final String keyPath,
-      final String authFile,
-      final HashicorpVault hashicorpVault) {
+      final Path tomlPath, final String authFile, final HashicorpNode hashicorpNode) {
     {
       try {
 
         final FileWriter writer = new FileWriter(tomlPath.toFile(), StandardCharsets.UTF_8);
         writer.write("[signing]\n");
         writer
             .append("type = \"hashicorp-signer\"\n")
-            .append("signing-key-path = \"" + keyPath + "\"\n")
-            .append("host = \"" + hashicorpVault.getIpAddress() + "\"\n")
-            .append("port = " + hashicorpVault.getPort() + "\n")
+            .append("signing-key-path = \"" + hashicorpNode.getSigningKeyPath() + "\"\n")
+            .append("host = \"" + hashicorpNode.getHost() + "\"\n")
+            .append("port = " + hashicorpNode.getPort() + "\n")
             .append("auth-file  = \"" + authFile + "\"\n")
-            .append("timeout = 500\n");
+            .append("timeout = 500\n")
+            .append("tls-enabled = " + hashicorpNode.isTlsEnabled() + "\n");
 
+        final String tlsConfig =
```

Comment:
move the tlsConfig creation into a new function

Reply:
this has been refactored slightly. Let me know if it needs more refactoring. resolving!